cmd/destroy/destroy.go

```diff
@@ -265,19 +266,27 @@ func (dc *destroyCommand) getDestroyer(ctx context.Context, opts *Options) (dest
 		}
 		runInRemote := !isRemote && (destroyImage != "" || opts.RunInRemote)
 
+		// destroy section should not be nil whether the command is executed remotely or locally
+		// if the command is executed remotely and destroy section is nil, this will cause a panic
+		// Hence consider the command locally for successful execution.
+		// This will make sure all the resources are deleted.
+		if runInRemote == true && manifest.Destroy == nil {
+			runInRemote = false
+		}
```

Comment:
```suggestion
		if manifest.Destroy == nil {
			runInRemote = false
		}
```

Reply:
maybe redundant if `runInRemote == false` but I think is clearer

Reply:
I tested it already. This will cause a panic to `okteto init` command. 

We need to have destroy section not as nil to execute the destroy command remotely. This destroy section was added automatically in case of valid manifests before checking whether we are running the command remotely.
Hence, if we find that even after executing the command remotely, the destroy section is nil, this means there is a chance of panic being raised. Hence consider this as the local case and set the value to false.

Reply:
**Update:** we've a separate issue for panic of _okteto init_ command. The above changes will cause no destroy panic. We can tackle the _okteto init_ panic in other issue.
cc @jLopezbarb

Reply:
Changes applied. Resolved!

Reply:
on second thought, let's avoid adding logic in this function, check my last comment :)